src/managers/KeybindManager.hpp

```diff
@@ -63,36 +62,36 @@ class CKeybindManager {
     std::list<SKeybind>                                               m_lKeybinds;
 
   private:
-    std::deque<xkb_keysym_t>  m_dPressedKeysyms;
-    std::deque<int>           m_dPressedKeycodes;
+    std::deque<xkb_keysym_t>     m_dPressedKeysyms;
+    std::deque<int>              m_dPressedKeycodes;
 
-    inline static std::string m_szCurrentSelectedSubmap = "";
+    inline static SubmapOptions* m_pCurrentSelectedSubmap = nullptr;
```

Comment:
I'd rather ditch the pointer. Whenever the vector gets realloc'd this will segfault.

I'd just keep the name and get a helper func like `getSubmapOptionsFromName`